src/transformers/generation/utils.py

```diff
@@ -2011,15 +2002,15 @@ def _prepare_cache_for_generation(
             elif generation_config.cache_implementation == "offloaded":
                 model_kwargs[cache_name] = OffloadedCache()
             elif generation_config.cache_implementation == "dynamic":
-                model_kwargs[cache_name] = DynamicCache(**dynamic_cache_kwargs)
+                model_kwargs[cache_name] = DynamicCache()
 
         # Use DynamicCache() instance by default. This will avoid back and forth from legacy format that
         # keeps copying the cache thus using much more memory
         else:
             model_kwargs[cache_name] = (
-                DynamicCache(**dynamic_cache_kwargs)
+                DynamicCache()
```

Comment:
isn't the previous version needed to propely use the new dynamic sliding window?

(= pass the config to trigger the dynamic sliding window)